### PR TITLE
TIC-99: Active/deactive a season of shows

### PR DESCRIPTION
**Pull request by HaliTran**
### Description
- Display whether a season is Active or Inactive. An active season is when all events part of that season is active. Otherwise, the season is Inactive.
- Implement checkbox to Activate or Deactivate a season. If checkbox is checked, all events within that season will be set to Active, and if unchecked, all events will be set to Inactive

### Risks
Minimal risks. May affect eve

### Validation
Verified in the database using PgAdmin and the swagger page that the events were updated correctly when a season was set to active and/or inactive. 

### Issue
https://pdx-hkaus.atlassian.net/jira/software/projects/TIC/boards/1?selectedIssue=TIC-99

### Operating System
M2 Macbook

### CODE REVIEW

File: client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx [hunk 1]
```diff
@@ -34,26 +50,91 @@ const ViewSeasonInfo = (props: ViewSeasonInfoProps) => {
...
+          >
+            {activeSeasonSwitch ? 'ACTIVE' : 'INACTIVE'}
+          </span>
+          <Tooltip
+            title={<p style={{fontSize: '1.2rem'}}>Edit</p>}
+            placement='top'
+            arrow
+          >
+            <button
+              className='flex justify-center items-center bg-gray-400 hover:bg-gray-500 disabled:bg-gray-500 text-white font-bold px-3 py-2 rounded-xl'
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx:69]: I'd replace the `px-3 py-2` with just a `w-11`. That way its a bit more square, and I think the other flex stuff will auto do the height.

File: client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx [hunk 2]
```diff
@@ -34,26 +50,91 @@ const ViewSeasonInfo = (props: ViewSeasonInfoProps) => {
...
+              </svg>
+            </button>
+          </Tooltip>
+          <Tooltip
+            title={<p style={{fontSize: '1.2rem'}}>Delete</p>}
+            placement='top'
+            arrow
+          >
+            <button
+              className='flex justify-center items-center bg-red-500 hover:bg-red-600 disabled:bg-gray-500 text-white font-bold px-3 py-2 rounded-xl'
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx:94]: Same suggestions as the above button/tooltip

File: client/src/components/Ticketing/ticketingmanager/Season/components/EventCard.tsx [hunk 3]
```diff
@@ -30,7 +31,7 @@ const EventCard = (props: EventCardProps) => {
     <div className='flex flex-col items-center bg-gray-200 rounded-xl border-8 border-solid border-zinc-300 mb-3 p-3 min-[768px]:flex-row min-[768px]:justify-between'>
       <article className='flex flex-col gap-3 items-center justify-center min-[768px]:flex-row'>
         <div id='event-image-container' className='w-36'>
-          <img
+          <SeasonImage
```
[the remark this answers is not in the document]
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Season/components/EventCard.tsx:34]: I created a SeasonImage component that is based off of the EventImage component when initially working on the season page. It has the same functionality has the EventImage one.

File: client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx [hunk 4]
```diff
@@ -34,26 +50,91 @@ const ViewSeasonInfo = (props: ViewSeasonInfoProps) => {
...
+        <div className='flex flex-col gap-2 tab:flex-row tab:flex-wrap'>
+          <span
+            className={`${
+              activeSeasonSwitch ? 'bg-green-100' : 'bg-red-100'
+            } py-2 px-8 rounded-lg font-medium`}
+          >
+            {activeSeasonSwitch ? 'ACTIVE' : 'INACTIVE'}
+          </span>
+          <Tooltip
+            title={<p style={{fontSize: '1.2rem'}}>Edit</p>}
```
[the remark this answers is not in the document]
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx:64]: Okay sounds good!

File: client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx (hunk 1 above)
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx:69]: Will do!

File: client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx (hunk 2 above)
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx:94]: Got it!

**Review by henrykaus — APPROVED** [review]

**Pull request merged**